src/pages/steps/CalendarStep/CalendarOpeninghoursModal.tsx

```diff
@@ -145,11 +145,15 @@ const CalendarOpeninghoursModal = ({
     idToChange: string,
   ) => {
     const checked = event.target.checked;
+    const daysOrder = Object.values(DaysOfWeek);
```

Comment:
We cannot rely on the order of the values from an object.
It may work now, but it may not work at some point according to the js spec.

keys and values in an object don't maintain a certain order, because they use a hashmap datastructure behind the scenes.
Since the codebase only uses the values anyways, never specific days we can refactor the DaysOfWeek to this:

```ts
const DaysOfWeek = [
  'monday',
  'tuesday',
  'wednesday',
  'thursday',
  'friday',
  'saturday',
  'sunday',
] as const;

type DayOfWeek = (typeof DaysOfWeek)[number];

```

Arrays do maintain their order of elements